srcpkgs/wezterm/REMOVE

```diff
@@ -0,0 +1,7 @@
+case "$ACTION" in
+pre)
+	# Remove compiled terminfo files.
+	rm -f usr/share/terminfo/w/wezterm.terminfo
```

Comment:
this should not be needed because it is owned by the package, and causes an error on uninstall:
```
ERROR: wezterm-20220624_1: failed to remove `./usr/share/terminfo/w/wezterm.terminfo': No such file or directory
```